db/schema.sql

```diff
@@ -9,19 +9,31 @@ CREATE TYPE VDAF_IDENTIFIER AS ENUM(
     'POPLAR1'
 );
 
+CREATE TYPE ROLE AS ENUM(
+    'COLLECTOR',
+    'CLIENT',
+    'LEADER',
+    'HELPER'
+);
+
 -- Corresponds to a PPM task, containing static data associated with the task.
 CREATE TABLE tasks(
     id                     BYTEA PRIMARY KEY,         -- 32-byte TaskID as defined by the PPM specification
-    ord                    BIGINT NOT NULL,           -- the order of this aggregator for this task; 0 is leader, 1 or larger is helper
+    role                   ROLE NOT NULL,             -- the role of this aggregator for this task
```

Comment:
I used an integer to allow for eventual expansion to multi-helper scenarios. That's probably preparing for a future that will never be, so this change makes sense.

But can we constrain this role value to be either `LEADER` or `HELPER`? These are specifically task parameters for `janus`, an aggregator, so the other role values do not make sense here.

Reply:
Sure, I removed the `COLLECTOR` and `CLIENT` values from the SQL enum. However now the SQL enum doesn't line up with `messages::Role`, so to reduce the risk of someone in the future assuming that the 0th value of the SQL enum is the same as the 0th value of `messages::Role`, I renamed the SQL enum to `AGGREGATOR_ROLE` and introduced a corresponding type in `datastore::models`.